test/spec/cloud-element-templates/properties-panel/ElementTemplatesPropertiesProvider.boolean.bpmn

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_1rghpyk" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.23.0-nightly.20240509" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.5.0">
+  <bpmn:process id="Process_1agx1l9" isExecutable="true">
+    <bpmn:task id="Task_1" zeebe:modelerTemplate="id" zeebe:modelerTemplateVersion="1715154218173">
+      <bpmn:extensionElements>
+        <zeebe:properties>
+          <zeebe:property name="myBooleanProp" value="true" />
+          <zeebe:property name="inputActive" value="test" />
+        </zeebe:properties>
+      </bpmn:extensionElements>
+    </bpmn:task>
+    <bpmn:task id="Task_2" zeebe:modelerTemplate="id" zeebe:modelerTemplateVersion="1715154218173">
+      <bpmn:extensionElements>
+        <zeebe:properties>
+          <zeebe:property name="myBooleanProp" value="false" />
```

Comment:
Let's test the general fix for `property` binding, i.e. `process#isExecutable`, too.

Reply:
OK